src/app-utils/register-shell-actions.ts

```diff
@@ -50,6 +57,59 @@ export const mailToAction = (contacts: unknown): MailToActionType => ({
 	disabled: isArray(contacts) && some(contacts, (contact) => !contact.address)
 });
 
+export type Recipient = {
+	email: string;
+	name: string;
+	carbonCopy?: boolean;
```

Comment:
An additional improvement could be to handle also the BCC, using an enumeration instead of a boolean flag